apps/web/src/pages/Skills/components/UpdateSkillShowcase.tsx

```diff
@@ -309,84 +278,57 @@ const UpdateSkillShowcase = ({
                         ),
                       }}
                     >
-                      {fields.length ? (
-                        fields.map((item, index) => (
-                          <Repeater.Fieldset
-                            key={item.id}
-                            index={index}
-                            total={fields.length}
-                            onMove={move}
-                            onRemove={remove}
-                            legend={
-                              <span
-                                data-h2-display="base(flex)"
-                                data-h2-justify-content="base(space-between)"
-                              >
-                                <span>
-                                  {getLocalizedName(
-                                    getSkill(item.skill)?.name ?? undefined,
-                                    intl,
-                                  )}
-                                </span>
-                                <span
-                                  data-h2-font-weight="base(400)"
-                                  data-h2-color="base(black.light)"
-                                >
-                                  {item.skillLevel
-                                    ? intl.formatMessage(
-                                        item.category ===
-                                          SkillCategory.Behavioural
-                                          ? getBehaviouralSkillLevel(
-                                              item.skillLevel,
-                                            )
-                                          : getTechnicalSkillLevel(
-                                              item.skillLevel,
-                                            ),
-                                      )
-                                    : getLocalizedName(null, intl)}
-                                </span>
-                              </span>
-                            }
-                          >
-                            <div>
-                              <p>
+                      {fields.map((item, index) => (
+                        <Repeater.Fieldset
+                          key={item.id}
+                          name="userSkills"
+                          index={index}
+                          total={fields.length}
+                          onMove={move}
+                          onRemove={remove}
+                          legend={
```

Comment:
Should we create an issue to apply the `onSave` function to this component so users can update their skills here?

Reply:
Good idea! Added by [9bf4f0c](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/9bf4f0c502671f42faec53630901ab7f55105235)

Reply:
Huh, okay so this sounded like a good idea but I think it may be out of scope. It works however, the create mutation is still run so instead of editing the existing one you end up creating a new one.

Sorry, since this is higher priority I'd say just back this out for now and we can add it in another issue.

Reply:
Let's just back this out then since this is high priority I can make an issue!

Reply:
Done! [2c00151](https://github.com/GCTC-NTGC/gc-digital-talent/pull/8181/commits/2c00151b2eaab2358759d1385cf6c557405382b1)